components/dashboard/src/projects/NewProject.tsx

```diff
@@ -193,33 +218,38 @@ export default function NewProject() {
 
     const renderSelectRepository = () => {
 
+        const noReposAvailable = reposInAccounts.length === 0;
+        const filteredRepos = Array.from(reposInAccounts).filter(r => r.account === selectedAccount && r.name.includes(repoSearchFilter));
         const icon = selectedAccount && accounts.get(selectedAccount)?.avatarUrl;
 
-        const renderRepos = () => (<div className="mt-10 border rounded-t-xl border-gray-100 flex-col">
-            <div className="px-8 pt-8 flex flex-col space-y-2">
-                <ContextMenu classes="w-full left-0 cursor-pointer" menuEntries={getDropDownEntries(accounts)}>
-                    <div className="w-full">
-                        <img src={icon} className="rounded-full w-6 h-6 absolute top-1/4 left-4" />
-                        <input className="w-full px-12 cursor-pointer font-semibold" readOnly type="text" value={selectedAccount || ""}></input>
-                        <img src={CaretDown} title="Select Account" className="filter-grayscale absolute top-1/2 right-3" />
+        const renderRepos = () => (<>
+            <div className={`mt-10 border ${isGitHub() ? "rounded-t-xl border-b-0" : "rounded-xl"} border-gray-100 flex-col`}>
+                <div className="px-8 pt-8 flex flex-col space-y-2">
```

Comment:
**question:** For some reason, the pre-selected account is a specific but maybe random group. Does it make sense to default to pre-selecting your own account?

Reply:
defaulting to own account sounds good.

Reply:
DEAL. Let's do this here or open a follow up issue to fine tune this later! Your call. 📞

Reply:
**fyi**: Opened follow up issue in https://github.com/gitpod-io/gitpod/issues/5330.

Reply:
**fyi**: Opened follow up issue in https://github.com/gitpod-io/gitpod/issues/5331.